File: SDKs/amazonaws.com/rds/2014-10-31/go/pkg/models/shared/dbclustersnapshotmessage.go
package shared

// DbClusterSnapshotMessage
//
//	Provides a list of DB cluster snapshots for the user as the result of a call to the <code>DescribeDBClusterSnapshots</code> action.
type DbClusterSnapshotMessage struct {
	DbClusterSnapshots []DbClusterSnapshot
	Marker             *string
}

File: SDKs/amazonaws.com/neptune/2014-10-31/typescript/src/sdk/models/shared/dbclustersnapshotmessage.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { DbClusterSnapshot } from "./dbclustersnapshot";



export class DbClusterSnapshotMessage extends SpeakeasyBase {
  @SpeakeasyMetadata({ elemType: DbClusterSnapshot })
  dbClusterSnapshots?: DbClusterSnapshot[];

  @SpeakeasyMetadata()
  marker?: string;
}
